deploy/kubernetes/console/templates/deployment.yaml

```diff
@@ -260,9 +264,13 @@ spec:
         app.kubernetes.io/name: "stratos"
         app.kubernetes.io/instance: "{{ .Release.Name }}"
         app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-        app.kubernetes.io/component: "console-mariadb"
+        app.kubernetes.io/component: "stratos-db"
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
         app: "{{ .Release.Name }}"
+    selector:
```

Comment:
This is under `spec.template` instead of `spec`. Does it still apply?

Reply:
I've sanity checked - should automate this check